shub/apps/api/management/commands/reset_container_limits.py

```diff
@@ -10,6 +10,7 @@
 
 from django.core.management.base import BaseCommand
 from shub.apps.main.models import Container
+from shub.apps.main.models import Shared
```

Comment:
This doesn't make sense - there is nothing from the main.models called "Shared." There is a file named shared.py that imports into the __init__.py so everything is importable from the main model (shub.apps.main.models). Can you share specifically the error you were running into?

Reply:
I think you actually meant to import `Collection` ?

Reply:
`from shub.apps.main.models import Container, Collection`

Reply:
True. Fixed in the last commit.